src/components/ExportGlStyleService.js

```diff
@@ -0,0 +1,78 @@
+goog.provide('ga_exportglstyle_service');
+
+goog.require('ga_browsersniffer_service');
+goog.require('ga_glstyle_service');
+
+(function() {
+
+  var module = angular.module('ga_exportglstyle_service', [
+    'ga_glstyle_service',
+    'ga_browsersniffer_service',
+    'pascalprecht.translate'
+  ]);
+
+  /**
+   * This service can be used to export a kml file based on some
+   * features on an ol map
+   */
+  module.provider('gaExportGlStyle', function() {
+    this.$get = function($translate, $window, $document, $http,
+        gaBrowserSniffer, gaGLStyle) {
+
+      var downloadUrl = this.downloadUrl;
+
+      var useDownloadService = function() {
+        if (gaBrowserSniffer.msie === 9 ||
+            gaBrowserSniffer.safari ||
+            !gaBrowserSniffer.blob) {
+          return true;
+        }
+        return false;
+      };
```

Comment:
when will this change? why not `var useDownloadService = !!(gaBrowserSniffer.msie === 9 || gaBrowserSniffer.safari || !gaBrowserSniffer.blob);`

Reply:
yes good point

Reply:
I know why know, because it 's easier to test.